README.md

```diff
@@ -28,6 +28,14 @@ The DevOps working group is an umbrella for multiple, more specific projects. Se
 - Communication: [Developer Experience](https://discuss.openedx.org/c/working-groups/dev-experience) working group forum category and #wg-developer-experience Slack channel.
 - Issue management: [Developer Experience working group](https://github.com/orgs/openedx/projects/37)
 
+### Large Instances working group
+
+- General information: [Large Instances workspace](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3655008783/Large+Instances+-+DevOps+Sub-Working+Group)
+- Communication: [bi-weekly meeting](https://discuss.openedx.org/t/deploying-open-edx-on-kubernetes-using-helm/8771), [DevOps](https://discuss.openedx.org/c/operators/7) forum category, [#wg-devops](https://openedx.slack.com/channels/wg-devops) Slack channel.
+- Issue management: [Large Instances working group](https://github.com/orgs/openedx/projects/42/views/2)
```

Comment:
FYI I created the project as a view on the DevOps project board: https://github.com/orgs/openedx/projects/42